layouts/partials/sidebar.html

```diff
@@ -34,3 +35,46 @@
     </li>
   {{ end }}
 </ul>
+
+<script>
```

Comment:
This script should probably be moved to the header/footer, and potential external?